File: easy/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
// The first submission -> 65ms, 42.1mb
var lengthOfLastWord = function(s) {
    let count = s.length
    let result = 0
    
    while (count--) {
        if (s[count] != " ") {
            result += 1
        } else if (result > 0) {
            break
        } else {
            // skip
        }
        
    }
    
    return result
};

File: easy/length-of-last-word.java
// The third submission -> 0ms, 40.4mb 
class Solution {
    public int lengthOfLastWord(String s) {
        char[] charArr = s.toCharArray();
        
        int count = charArr.length;
        int result = 0;

        while (count-- > 0) {
            if (charArr[count] != ' ') {
                result++;
            } else if (result > 0) {
                return result;
            }
        }

        return result;
    }
}
